docs/docs/platform/quickstarts/nextjs.mdx

```diff
@@ -0,0 +1,744 @@
+---
+title: 'Quickstart: Next.js'
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Quickstart: Next.js
+
+## Introduction
+
+This quickstart guide provides the steps you need to build a simple Next.js app
+powered by Nhost for the backend. It includes:
+
+- Database: [PostgreSQL](https://www.postgresql.org/)
+- Instant GraphQL API: [Hasura](https://hasura.io/)
+- Authentication: [Hasura Auth](https://github.com/nhost/hasura-auth/)
+- Storage: [Hasura Storage](https://hub.docker.com/r/nhost/hasura-storage)
+
+By the end of this guide, you'll have a full-stack app that allows users to log
+in to access a protected dashboard and update their profile information.
+
+## Prerequisites
+
+Before getting started, let's make sure that your development environment is
+ready.
+
+You'll need **Node.js** version 10.13 or later: [install it from here](https://nodejs.org/en/).
```

Comment:
I think we need a more recent Node version.
Nextjs requires [v12](https://nextjs.org/docs#system-requirements) and we run our tests in [v14 and v16](https://github.com/nhost/nhost/blob/main/.github/workflows/tests.yaml) until now

Reply:
Updated.